src/Home/ProximityTracingActivationStatus.tsx

```diff
@@ -39,9 +46,16 @@ export const ProximityTracingActivationStatus: FunctionComponent = () => {
     )
   }
 
-  const handleOnPressFix = () => {
-    exposureNotifications.request()
-    if (isNotAuthorized) {
+  const handleOnPressFix = async () => {
+    try {
+      await exposureNotifications.request()
+
+      const isNotENEnabled = status !== ENPermissionStatus.ENABLED
+
+      if (isNotENEnabled) {
+        showNotAuthorizedAlert()
+      }
+    } catch (error) {
```

Comment:
Why is it not sad that `error` is unused here?

Reply:
in short, it's not a language feature.

Here's any issue that touches a little on the why catches dont have type annotations: https://github.com/Microsoft/TypeScript/issues/20024

Reply:
removing the unused error parameter altogether though!